packages/@local/eslint-config/index.js

```diff
@@ -303,7 +303,7 @@ module.exports = {
     {
       files: ["site/**"],
       parserOptions: {
-        tsconfigRootDir: `${__dirname}/site`,
+        tsconfigRootDir: `${__dirname}/../../../site`,
```

Comment:
Regression from #508, spotted by @yusufkinatas. We resolved it locally in DMs. Attaching the fix here as it does not seem worth a separate PR.